server/consumer.go

```diff
@@ -1882,6 +1882,25 @@ func (o *consumer) checkPendingRequests() {
 	o.prm = nil
 }
 
+// This will release any pending pull requests if applicable.
+// Should be called only by the leader being deleted or stopped.
+// Lock should be held.
+func (o *consumer) releaseAnyPendingRequests() {
+	if o.mset == nil || o.outq == nil || o.waiting.len() == 0 {
+		return
+	}
+	hdr := []byte("NATS/1.0 409 Consumer Deleted\r\n\r\n")
```

Comment:
I guess we don't need to send the pending msgs/bytes in this case since the consumer is deleted.

Reply:
I think that is correct, but should we add it?